webpack.config.js

```diff
@@ -60,6 +63,20 @@ module.exports = (env, argv) => {
     fileLoaderPath = "file-loader?name=[name].[ext]";
   }
 
+  // Create release if sentry config is set
+  let sentryPlugin;
+  if (
+    process.env.SENTRY_ORG &&
+    process.env.SENTRY_PROJECT &&
+    process.env.SENTRY_DSN &&
+    process.env.SENTRY_AUTH_TOKEN
+  ) {
+    sentryPlugin = new SentryWebpackPlugin({
+      include: "./build/dashboard/",
```

Comment:
Is release number from package.json added automatically?
Not sure if it adds SHA commit or version in our case ( https://github.com/getsentry/sentry-webpack-plugin#options )

Reply:
It's not added automatically, let's come back to this topic once we have automated release bumps. For now the SHA commit should do the trick ✨